src/settings/OrderTemplates/OrderTemplateView/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as OrderTemplateView } from './OrderTemplateView';
```

Comment:
@NikitaSedyx Could you pleas check this index? Such exports looks strange to me, but maybe it's me